extensions/sql-database-projects/package.json

```diff
@@ -254,6 +267,14 @@
         {
           "command": "sqlDatabaseProjects.createProjectFromDatabase"
         },
+        {
+          "command": "sqlDatabaseProjects.updateProjectFromDatabase",
+          "when": "false"
+        },
+        {
+          "command": "sqlDatabaseProjects.getProjectDatabaseSchemaProvider",
```

Comment:
this command isn't needed anymore now that it got added to sqldbproj.IExtension, right?